coalib/bearlib/languages/documentation/DocumentationExtraction.py

```diff
@@ -162,14 +162,21 @@ def _extract_doc_comment(content, line, column, markers):
                     line, column and the extracted documentation. If not
                     matched, returns None.
     """
-    if markers[1] == '':
-        # Extract and align to start marker.
-        return _extract_doc_comment_simple(content, line, column, markers)
-    elif markers[1] == markers[2]:
-        # Search for the each-line marker until it runs out.
-        return _extract_doc_comment_continuous(content, line, column, markers)
-    else:
-        return _extract_doc_comment_standard(content, line, column, markers)
+    # This ignores fancy docstrings.
+    valid_docstring_regex = '\s*[' + markers[0] + '].*'
```

Comment:
Ok so it goes like this `marker_dict` is specifically made for `begin_regex` to find the `marker[0]` exactly as it is.
As `marker[0]` is the key of `marker_dict` which will return `markers`. eg`marker_dict` for python is `['"""':['"""','','"""']]`.
Hence, this is the reason why I can't change/tamper `begin_regex`. It would require a major change so as to do it(which I think it should not be removed as it has its importance). look at this [for more info.](https://github.com/damngamerz/coala/blob/690ab278c331ffa2c6d53179e5d010b23c9107f6/coalib/bearlib/languages/documentation/DocumentationExtraction.py#L230)